src/_includes/publication.html

```diff
@@ -141,7 +141,9 @@ <h5 class="g-color-black g-font-weight-600">{{ page.author | author_links}}</h5>
         {% assign cta_href = "/services/software-modernisation/" | prepend: site.baseurl %}
         {% capture cta_text %}{% t software-modernisation.in-page-banner-publications-link %}{% endcapture %}
         {% capture title %}{% t software-modernisation.in-page-banner-publications-title %}{% endcapture %}
-        {% include in_page_banner.html background_image=background_image class="container section g-pt-200 g-pb-100" cta_href=cta_href cta_text=cta_text description=description title=title %}
+        {%- capture hubspot-cta -%}{%- include soft_mod_husbpot_in_page_banner--blog_cta.liquid -%}{%- endcapture -%}
+
+        {% include in_page_banner.html background_image=background_image class="container section g-pt-200 g-pb-100" cta_href=cta_href cta_text=cta_text description=description title=title hubspot-cta=hubspot-cta %}
```

Comment:
Perhaps I'm reading it badly but I don't see where `cta_href` and `cta_text` are being used. Can we delete their assignments?

Reply:
Good spot. This has been amended in the following commit [30a2d26](https://github.com/codurance/site/commit/30a2d26e08bed88020509062ac00293618f0921d)